kernel/fablib/ssh.go

```diff
@@ -131,29 +131,41 @@ func RemoteConsole(factory SshConfigFactory, cmd string) error {
 }
 
 func RemoteExec(sshConfig SshConfigFactory, cmd string) (string, error) {
+	return RemoteExecSeq(sshConfig, cmd)
+}
+
+func RemoteExecSeq(sshConfig SshConfigFactory, cmds ...string) (string, error) {
```

Comment:
Maybe `RemoteExecAll`?

Reply:
Yeah, there's no corresponding RemoteExecParallel, so the Seq doesn't really make sense. 

I like`RemoteExecEach`, how do you feel about that vs `RemoveExecAll`?

Reply:
That's fine. To be super nit-picky... "each" kind of implies that you're passing something a function, IMO. Things called "each" are typically a way of exposing a loop to some extensible component. In this case we're just running a group of commands sequentially, so you're passing in an array to be iterated, and `RemoteExec` is doing something with that array.

So, I'm not strongly opposed or anything, but also doesn't seem like an "each" thing to me.

Reply:
Your semantic nuance argument resonates. I'll change it to ``RemoveExecAll`